protobuf/benchmarks/README.md

```diff
@@ -8,25 +8,32 @@ JSON formats, and hash code generation for messages.
 
 ## Prerequisites
 
-Before running benchmarks you first need to compile all involved
-protos, which requires installing `protoc` and
-[dart-protoc-plugin](https://github.com/dart-lang/dart-protoc-plugin).
+Before running benchmarks you first need to compile all involved protos, which
+requires installing `protoc` and [protoc_plugin](../../protoc_plugin).
 
 ```console
-$ cd benchmarks
 $ ./compile-protos.sh
-# This would produce benchmarks/temp folder with output of protoc.
 ```
 
+This produces temp folder with output of protoc.
```

Comment:
Actually that seems kind of non-standard. Should we not generate into `lib/` or something?

(Doesn't have to be this PR)